web/templates/settings/emoji_settings_tip.hbs

```diff
@@ -1,10 +1,10 @@
 {{#if is_guest}}
     <div class='tip'>{{t "Guests cannot edit custom emoji." }}</div>
-{{else if (eq realm_add_custom_emoji_policy policy_values.by_admins_only.code) }}
+{{else if (eq realm_can_add_custom_emoji_group_name "role:administrators") }}
     <div class='tip'>{{t "This organization is configured so that only administrators can add custom emoji." }}</div>
-{{else if (eq realm_add_custom_emoji_policy policy_values.by_moderators_only.code)}}
+{{else if (eq realm_can_add_custom_emoji_group_name "role:moderators")}}
     <div class='tip'>{{t 'This organization is configured so that administrators and moderators can add custom emoji.'}}</div>
-{{else if (eq realm_add_custom_emoji_policy policy_values.by_full_members.code)}}
+{{else if (eq realm_can_add_custom_emoji_group_name "role:fullmembers")}}
     <div class='tip'>{{t 'This organization is configured so that full members can add custom emoji.'}}</div>
 {{else}}
     <div class='tip'>{{t "This organization is configured so that any member of this organization can add custom emoji." }}</div>
```

Comment:
We may find we want to replace this with a single string; the complexity won't be worth it once we allow normal groups to be used...